gofig.go

```diff
@@ -68,6 +102,8 @@ func New(errHandling ErrHandling) *Gofig {
 func SetConfigFileFlag(name string, desc string) {
 	gf.SetConfigFileFlag(name, desc)
 }
+
+// SetConfigFileFlag adds a config file flag
```

Comment:
Add documentation on each of these so that it's properly parsed by godoc, at the expense of repeated comment lines.

Also see:
https://goreportcard.com/report/github.com/curvegrid/gofig